src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -49,15 +52,18 @@ public BtConnectorHelper() {
         this.context = jxcore.activity.getBaseContext();
     }
 
-
     public BTConnector.WifiBtStatus Start(String peerName,int port){
```

Comment:
Here is a fun example of why sometimes you have to use synchronized methods. Ask yourself the following question - what happens if start and stop both get called at the same time from different threads?

But here is a better question - is it possible for start and stop to be called at the same time from different threads? I don't think so. But we should be sure. Here is my reasoning - The only entity that should EVER call BtConnectorHelper is JXcoreExtension and JXcore guarantees that JXcoreExtension will always be called on the same thread (at least I think it guarantees that, we should check with Oguz). In that case methods like start and stop can only be called on a single thread since only JXcoreExtension can call them. If that is all true then this is worth some kind of comment at the top of the file pointing out that we are depending on the fact that other than in callbacks all the methods in BtConnectorHelper will only be called by JXcoreExtension and JXcoreExtension will only be called from the same thread.